simulation_utils/ParameterParser.cpp

```diff
@@ -77,14 +81,21 @@ void Simulation_Parameters::getParamsFromFileAndSample(std::string filename) {
 
 VectorXd Simulation_Parameters::initializeSatellite(double epoch)
 {
-    VectorXd State(13+num_RWs);
+    VectorXd State(19+num_RWs);
 
     Vector6 KOE {semimajor_axis, eccentricity, inclination, RAAN, AOP, true_anomaly};
     Vector6 CartesianState = KOE2ECI(KOE, epoch);
     State(Eigen::seqN(0,6)) = CartesianState;
     State(Eigen::seqN(6,4)) = initial_attitude;
     State(Eigen::seqN(10,3)) = initial_angular_rate;
-    State(Eigen::seqN(13, num_RWs)).setZero();
+    // Sun Position [m]
+    State(Eigen::seqN(13, 3)) = Eigen::VectorXd::Zero(3); //sun_position_eci(epoch);
+    // Magnetic Field [T]
+    // Vector3 Bfield = MagneticField(CartesianState(Eigen::seqN(0, 3)), epoch);
```

Comment:
Why is this commented out? This is the best way to load it into the state

Reply:
wasnt working. Need to recheck

Reply:
I dont know how to fix this. Theres some spice bug I dont know how to solve. I would ask you to help fix it whenever you find the time